distributed/protocol/utils.py

```diff
@@ -126,14 +129,13 @@ def unpack_frames(b):
     --------
     pack_frames
     """
-    (n_frames,) = struct.unpack("Q", b[:8])
-
-    frames = []
-    start = 8 + n_frames * 8
-    for i in range(n_frames):
-        (length,) = struct.unpack("Q", b[(i + 1) * 8 : (i + 2) * 8])
-        frame = b[start : start + length]
-        frames.append(frame)
-        start += length
+    fmt = "Q"
+    fmt_itemsize = struct.calcsize(fmt)
```

Comment:
I was not familiar with `calcsize`, this and the logic below is the equivalent of [:8], correct ?

Reply:
Yeah that's right.